frontend/src/components/Layout/Header/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import './styles.css';
+import logo from '../../../img/logo/logo_seul_couleur.png';
+
+export const Header: React.FC = () => {
+  let appUrl = 'https://' + window.location.hostname + '/app/';
```

Comment:
Be careful here you are declaring the appUrl, which is APLOSE URL but this is the link to go to OSmOSE website. You should remove the `/app` to access OSmOSE website.

Also, `let` is used for variables that can be reassigned, for the variables that are assigned only one time like here, we would prefer `const`declaration